src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Media/PixelFormat.cs

```diff
@@ -71,68 +74,63 @@ internal PixelFormatChannelMask(byte[] mask)
 
         /// <summary>
         /// The bitmask for a color channel
-        /// It will never be greater then 0xffffffff
+        /// It will never be greater than 0xFFFFFFFF.
         /// </summary>
         public IList<byte> Mask
         {
             get
             {
-                return _mask != null ? new PartialList<byte>((byte[])_mask.Clone()) : null;
+                return _mask?.ToArray();
```

Comment:
I don't think returning a mutable IList here would break anything but I don't see the value in changing the existing behavior. Changing it to returning a mutable IList also means the API will now need to always return a mutable IList because in the future code could depend on the fact that they can modify the IList. Going non-mutable -> mutable is not a breaking change but the inverse is a breaking change.

This is what I would do (I also changed it to use ReadOnlyCollection here since it's built-in and we don't need the "partial list" feature of PartialList):
```suggestion
                return _mask != null ? new ReadOnlyCollection<byte>(_mask.AsSpan().ToArray()) : null;
```

Reply:
It's a good point that in the future you would have this problem, indeed.

However, in that case, I do not understand why would you make a copy of the array if you're a returning readonly wrapper? Guess that's an oversight.